src/main/java/mate/hwdao/controllers/AddCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.hwdao.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.hwdao.Main;
+import mate.hwdao.lib.Injector;
+import mate.hwdao.model.Car;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.service.CarService;
+import mate.hwdao.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+
```

Comment:
I would still recommend to remove empty lines after class declarations
as you wrote: A single blank line may also appear anywhere it improves readability, for example between statements to organize the code into logical subsections.

this line does not improve readability and is not placed between statements